src/example-types/metricGroups/groupStatUtils.js

```diff
@@ -1,30 +1,30 @@
-let { statsAggs, simplifyBuckets } = require('../../utils/elasticDSL')
-let { getStats } = require('./stats')
+import { statsAggs, simplifyBuckets } from '../../utils/elasticDSL.js'
+import stats from './stats.js'
+
+let { getStats } = stats
```

Comment:
can't we destructure `getStats` off the import?

Reply:
This is something that commonjs was looser on. You could do `module.exports = { foo: 'bar' }` and then `let { foo } = require('file')` or `let mod = require('file').foo`. ESM is more strict though so `export default { foo: 'bar' }` corresponds to `import mod from 'file'` and `export { foo: 'bar' }` to `import { foo } from 'file'`